src/components/Buttons.tsx

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable */
+import React from 'react';
+
+interface Props {
+  children: React.ReactNode;
+  style?: any;
+  onClick?: () => void;
+  variant: string;
+  size: string;
+}
+const Button: React.FC<Props> = ({
+  children,
+  style,
+  onClick,
+  variant = 'default',
+  size = 'md',
+  ...rest
+}) => (
+  <button
+    className={`btn ${variant} ${size} ${style}`}
+    onClick={onClick}
+    {...rest}
+  >
+    {children}
+  </button>
```

Comment:
Good job on defininng the Props object and passing it as a functional component. I'm wondering what would happen if invalid values are given for example, instead of passing `md` for size, someone passes `mf`. Are such cases handled?

Reply:
actually, I didn't think about that but there is another way to add a style to the button.